src/js/lib/content/src/docsStyle.css

```diff
@@ -0,0 +1 @@
d{margin-left:0;border:2px solid #4b5964;display:inline-table;border-radius:16px;padding:2px 6px;line-height:1.35;box-decoration-break:clone;-webkit-box-decoration-break:clone}.light-theme .doc .tag .private{margin-left:0;border:2px solid #4b5964;display:inline-table;border-radius:16px;padding:2px 6px;line-height:1.35;box-decoration-break:clone;-webkit-box-decoration-break:clone}.light-theme .doc .tag .textonly{margin-left:0;border:2px solid #4b5964;display:inline-table;border-radius:16px;padding:2px 6px;line-height:1.35;box-decoration-break:clone;-webkit-box-decoration-break:clone}.light-theme .doc .tag .unstable{margin-left:0;border:2px solid #ec2915;color:#ec2915;display:inline-table;border-radius:16px;padding:2px 6px;line-height:1.35;box-decoration-break:clone;-webkit-box-decoration-break:clone}.light-theme .doc .tag .upcoming{margin-left:0;border:2px solid #4b5964;display:inline-table;border-radius:16px;padding:2px 6px;line-height:1.35;box-decoration-break:clone;-webkit-box-decoration-break:clone}.light-theme .doc .tag .alias{margin-left:0;border:2px solid #4b5964;display:inline-table;border-radius:16px;padding:2px 6px;line-height:1.35;box-decoration-break:clone;-webkit-box-decoration-break:clone}.light-theme .doc .tag .version{margin-left:0;border:2px solid #4b5964;display:inline-table;border-radius:16px;padding:2px 6px;line-height:1.35;box-decoration-break:clone;-webkit-box-decoration-break:clone}.light-theme .doc .atom{font-size:20px;line-height:1.35;margin-top:2rem;color:#4b5864}.light-theme .doc .atom .name{font-weight:700;color:#2a394a}.light-theme .doc .method{font-size:20px;line-height:1.35;margin-top:2rem;color:#4b5864}.light-theme .doc .method .name{font-weight:700;color:#2a394a}.light-theme .doc .method .argument{opacity:.5}.light-theme .doc .method .argument.default{opacity:.5}.light-theme .doc .submodule{font-size:20px;line-height:1.35;margin-top:2rem;color:#4b5864}.light-theme .doc .submodule .name{font-weight:700;color:#2a394a}.light-theme .doc .toc h2{margin-top:0;font-size:24px;line-height:1.35}.light-theme .doc .toc .methods{margin-top:2rem}.light-theme .doc .toc .atom{font-size:16px;line-height:1.4;margin-top:0}.light-theme .doc .toc .atom .name{font-weight:500}.light-theme .doc .toc .method{font-size:16px;line-height:1.4;margin-top:0}.light-theme .doc .toc .method .name{font-weight:500}.light-theme .doc .breadcrumb-panel{padding-top:.75rem;padding-bottom:.75rem;position:-webkit-sticky;position:sticky;top:0;z-index:10;background-color:#2077d0}.light-theme .doc .breadcrumbs{color:hsla(0,0%,100%,.9)}.light-theme .doc .breadcrumbs .separator{margin-right:.5rem;margin-left:.5rem;font-size:12px}.light-theme .doc .breadcrumbs .browse-submodules{margin-left:0;background-color:rgba(235,241,251,.2);display:inline-table;border-radius:8px;padding:2px 6px;line-height:1.35;box-decoration-break:clone;-webkit-box-decoration-break:clone}@media (min-width:768px){.light-theme .doc h1{font-size:48px;line-height:1.25}.doc .toc{display:block}}
```

Comment:
Can you properly format this so it is readable?

Reply:
Well it is generated from the less file and it is minified - that's why it is unreadable :P